HHS/Head-Start-TTADP
# [TTAHUB-1991] Refactor permissions and roles in the admin

## Description of change

- Remove training report flag from frontend and backend
- Rearrange permissions on the admin UI and add an "all regions option"

## How to test
- Check that the admin UI has been arranged appropriately. 
- Check that you can perform CRUD operations on roles and permissions
- Check that you can set a user to have a permission in all regions*
- Check that the correct people can still view training reports

\* **I understand that "All regions" does not stay selected if you navigate away or even if you load up a user where they nominally have all regions selected. Hopefully we can roll with this as a convenience for admins.** 

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1991


## Checklists

### Every PR



### Production Deploy


### After merge/deploy
